java/connector-node/risingwave-connector-service/src/main/resources/debezium.properties

```diff
@@ -12,6 +12,3 @@ interval.handling.mode=string
 max.batch.size=${debezium.max.batch.size:-1024}
 max.queue.size=${debezium.max.queue.size:-8192}
 time.precision.mode=adaptive_time_microseconds
-# The maximum number of retries on connection errors before failing
```

Comment:
Revert back to default setting: `-1` (it is a new config after we bumped debezium), we can modify it if we have more usecases.